pkg/cmd/plugin_list.go

```diff
@@ -51,7 +58,7 @@ func newPluginListCmd(out io.Writer) *cobra.Command {
 			signingInfo := plugin.GetSigningInfoForPlugins(plugins)
 
 			table := uitable.New()
-			table.AddRow("NAME", "VERSION", "TYPE", "APIVERSION", "PROVENANCE", "SOURCE")
+			table.AddRow("NAME", "VERSION", "LATEST", "TYPE", "APIVERSION", "PROVENANCE", "SOURCE")
```

Comment:
Is it a breaking change to add a new column? I'm thinking about script that parse the output? Maybe it should be scopped behind `o.showOutdated` ?